oeis.py

```diff
@@ -413,6 +413,23 @@ def A007953(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence
 
 
+@oeis
+def A000120(start: int = 0, limit: int = 20) -> Collection[int]:
+    """1's-counting sequence: number of 1's in binary
+    expansion of n (or the binary weight of n).
+    """
+    sequence = []
+
+    for n in range(start, start + limit):
+        count = 0
```

Comment:
This line can be removed safely, it is not used.

Reply:
I insist, this line is unused, you can remove it.

This also means it looks like:
```python
acc = []
for something in a_collection:
    acc.append(a_modification_on(something))
```

Which is such a common pattern in Python we have a dedicated syntax for this: list comprehensions:
```python
acc = [a_modification_on(something) for something in a_collection]
```

So if you want, your function can even start with `return` and span on a single line of code.